ddtrace/contrib/consul/patch.py

```diff
@@ -43,9 +42,10 @@ def trace_func(wrapped, instance, args, kwargs):
             return wrapped(*args, **kwargs)
 
         path = kwargs.get('key') or args[0]
+        resource = '%s %s' % (name.upper(), path)
 
-        with pin.tracer.trace(name, service=pin.service, resource=path) as span:
-            span.set_tag('consul.key', path)
+        with pin.tracer.trace(consulx.CMD, service=pin.service, resource=resource) as span:
+            span.set_tag(consulx.KEY, path)
```

Comment:
we _could_ set a tag here as well for `set_tag('consul.command', 'GET')`

would allow people to filter to just `GET` commands in trace search and analytics

Reply:
Done